src/explanations/dai.yaml

```diff
@@ -0,0 +1,82 @@
+    The MCD_DAI_JOIN adapter contract in the Maker Protocol is authorized to call the mint and burn functions of the DAI token in order to increase or decrease the total supply of DAI. These functions are also used when users deposit or withdraw DAI from the Dai Savings Rate contract.
+
+
+    ### Transparency and Audits
+
+
+    MakerDAO and the DAI stablecoin are designed to be transparent and auditable. The smart contracts that govern the Collateralized Debt Positions (CDPs) and the minting and burning of DAI are open-source, which allows anyone to review the code and understand how the system works.
+
+
+    In addition, MakerDAO has undergone several independent audits by reputable firms, which have found that the system is secure and functioning as intended. In 2019, Callisto Network Security conducted a security audit of the Dai Token (DAI) smart contract which identified four issues, including three of low severity and one of owner privileges (which could potentially be risky for investors). No critical security issues were discovered. The audit specifically identified known vulnerabilities of ERC-20 tokens, the ability for the contract owner to block transferring, a missing event in ERC20 compliance, and the need to check input addresses for empty values. Overall, the audited smart contract was determined to be safe to deploy, with only low severity issues found during the audit. [Here](https://callisto.network/dai-token-dai-security-audit/) is the complete report.
+
+
+    ### Backing of DAI
+
+
+    All DAI is backed by surplus collateral that has been escrowed in audited and publicly viewable Ethereum smart contracts. This means that anyone with an internet connection can monitor the health of the system at any time by visiting [daistats.com](https://daistats.com/).
+
+
+    The value of the collateral used to mint DAI must always exceed the value of the DAI itself, a process known as overcollaterization. This is to ensure that the collateral can cover the value of the DAI in the event that the market value of the collateral falls. If the value of the collateral falls below the value of the DAI, the vault account may be liquidated, with a ratio of 1.5 collateral to 1 DAI.
+
+
+    To ensure that undercollateralized accounts have sufficient funds to be liquidated, the MakerDAO system offers "keeper" accounts. Keepers are special user accounts that stake their cryptocurrency holdings to provide liquidity to pay off the collateral of vault accounts that lose value over time. This helps to maintain a steady supply of DAI in circulation to match the value of one US dollar.
+
+
+    ### Issues and Hacks
+
+
+    Like any complex system, MakerDAO and DAI have experienced some issues and challenges. In 2020, the value of Ethereum plummeted, which caused many CDPs to be liquidated and resulted in a significant amount of DAI being minted. This led to a temporary increase in the supply of DAI, which caused its value to drop below one US dollar.
```

Comment:
We should capture and include more specifics about this event
https://medium.com/@whiterabbit_hq/black-thursday-for-makerdao-8-32-million-was-liquidated-for-0-dai-36b83cac56b6